### PR TITLE
20 check ttl of session key in test backend

### PR DESCRIPTION
**Pull request by AlexNi245**

### CODE REVIEW

File: packages/backend/src/auth.ts [hunk 1]
```diff
@@ -36,68 +37,80 @@ export default () => {
...
+                    req.app.locals.db.setSession,
+                    req.body.signature,
+                    account,
+                );
+
+                res.json({
+                    token,
+                });
+            } catch (e) {
+                console.log(e);
```
Comment (hai-ko) [inline packages/backend/src/auth.ts:109]: Remove console.log

File: packages/backend/src/delivery.ts [hunk 2]
```diff
@@ -76,6 +76,7 @@ export default () => {
 
                 res.json(newMessages);
             } catch (e) {
+                console.log(e);
```
Comment (hai-ko) [inline packages/backend/src/delivery.ts:79]: remove console.log

File: packages/backend/src/messaging.test.ts [hunk 3]
```diff
@@ -28,7 +28,7 @@ describe('Messaging', () => {
         });
     });
     describe('submitMessage', () => {
-        it('returns success if schema is valid', (done: any) => {
+        it.only('returns success if schema is valid', (done: any) => {
```
Comment (hai-ko) [inline packages/backend/src/messaging.test.ts:31]: remove `.only`

File: packages/backend/src/persistance/messages/deleteExpiredMessages.test.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import { getDatabase, getRedisClient, IDatabase, Redis } from '../getDatabase';
-describe('Delete Expired messages', () => {
+describe.skip('Delete Expired messages', () => {
```
Comment (hai-ko) [inline packages/backend/src/persistance/messages/deleteExpiredMessages.test.ts:2]: Is the `.skip` on puropse

File: packages/backend/src/rpc/rpc-proxy.test.ts [hunk 5]
```diff
@@ -11,7 +11,7 @@ const RECEIVER_ADDRESS = '0xDd36ae7F9a8E34FACf1e110c6e9d37D0dc917855';
 
 const keyPair = Lib.crypto.createKeyPair();
 
-describe('rpc-Proxy', () => {
+describe.only('rpc-Proxy', () => {
```
Comment (hai-ko) [inline packages/backend/src/rpc/rpc-proxy.test.ts:14]: remove `.only`

File: packages/lib/package.json [hunk 6]
```diff
@@ -39,7 +39,7 @@
...
+        "build:schema": "yarn ts-json-schema-generator -f tsconfig.json --path src/lib/Messaging.ts --type Message -o src/schema.json --no-type-check && yarn ts-json-schema-generator -f tsconfig.json --path lib/src/delivery/Messages.ts --type Acknoledgment -o src/delivery/schema/Acknoledgment.schema.json --no-type-check && yarn ts-json-schema-generator -f tsconfig.json --path lib/src/messaging/Messaging.ts --type EncryptionEnvelop -o src/messaging/schema/EncryptionEnvelop.schema.json --no-type-check  && yarn ts-json-schema-generator -f tsconfig.json --path lib/src/delivery/Delivery.ts --type DeliveryServiceProperties -o src/delivery/schema/DeliveryServiceProperties.schema.json --no-type-check && yarn ts-json-schema-generator -f tsconfig.json --path lib/src/delivery/Session.ts --type Session -o src/delivery/schema/Session.schema.json --no-type-check",
```
Comment (hai-ko) [inline packages/lib/package.json:42]: do we already have an issue for making this shorter?

**Review by hai-ko — CHANGES_REQUESTED** [review]

File: packages/lib/package.json (hunk 6 above)
Comment (AlexNi245) [reply packages/lib/package.json:42]: I can create one

**Review by hai-ko — APPROVED** [review]

**Pull request merged**